plugins/quickinspector/quickoverlay.cpp

```diff
@@ -50,6 +50,50 @@ extern Q_GUI_EXPORT QImage qt_gl_read_framebuffer(const QSize &size,
 QT_END_NAMESPACE
 
 namespace GammaRay {
+
+class QQuickItemPropertyCache {
+public:
+    static const QQuickItemPropertyCache &getPropertyCache(QQuickItem *item)
+    {
+        static QHash<QByteArray, QQuickItemPropertyCache> s_cache;
+        const auto meta = item->metaObject();
+        const auto className = QByteArray(meta->className());
```

Comment:
I have not idea if a QMetaObject instance classname data is still valid after the object destruction.
You would have to read the (generated) code for knowing that.
If it's still valid then of course you don't have to track objects for deletion.